recipes/usockets/all/test_package/conanfile.py

```diff
@@ -1,17 +1,26 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
 import os
 
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```

Comment:
I would suggest saving time and skipping the test package updates since the recipe is still only building with the legacy tools

Reply:
I agree with you.
I revert test_package/conanfile.py and remove test_v1_package.